src/app/CreateRecording/CreateRecording.tsx

```diff
@@ -62,24 +63,32 @@ const Comp: React.FunctionComponent< RouteComponentProps<{}, StaticContext, Crea
   const context = React.useContext(ServiceContext);
   const notifications = React.useContext(NotificationsContext);
   const history = useHistory();
+  const addSubscription = useSubscriptions();
 
   const [activeTab, setActiveTab] = React.useState(0);
 
-  const handleSubmit = (command: string, args: string[]): void => {
-    const id = context.commandChannel.createMessageId();
-    context.commandChannel.onResponse(command).pipe(
-      filter(m => m.id === id),
-      first(),
-      )
-      .subscribe((msg) => {
-          if (msg.status === 0) {
-            notifications.success('Recording created');
-            history.push('/recordings');
-          } else {
-            notifications.danger(`Request failed (Status ${msg.status})`, msg.payload)
```

Comment:
Would it make sense to notify on failure in the new subscription below? I don't see handling of that at the moment.

Reply:
There's a `tap` in the `context.api.createRecording` (Api.service.tsx line 14) that intercepts the response and displays the equivalent notifications that were removed here, so the handling is in the end unchanged, just done somewhere else.